src/Analyzers/Core/Analyzers/ValidateFormatString/ValidateFormatStringOption.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.ValidateFormatString
+{
+    internal class ValidateFormatStringOption
```

Comment:
Split into a separate file from `ValidateFormatStringOptionProvider`, latter is only defined in Workspaces layer.